From 1d8b33e10e2de92369f1e99d3d4694fd15b557ec Mon Sep 17 00:00:00 2001
From: Morten Welinder <terra@gnome.org>
Date: Wed, 25 May 2011 02:10:43 +0000
Subject: Sheet: fix chain in constructed method.

---
diff --git a/src/sheet.c b/src/sheet.c
index 0c1ab57..c44e21f 100644
--- a/src/sheet.c
+++ b/src/sheet.c
@@ -702,7 +702,8 @@ gnm_sheet_constructed (GObject *obj)
 
 	sheet_scale_changed (sheet, TRUE, TRUE);
 
-	parent_class->constructed (obj);
+	if (parent_class->constructed)
+		parent_class->constructed (obj);
 }
 
 static guint
--
cgit v0.9
