--- gnumeric-1.8.4/gnumeric.desktop.in.desktop	2009-04-13 10:22:21.000000000 +0530
+++ gnumeric-1.8.4/gnumeric.desktop.in	2009-04-13 10:22:51.000000000 +0530
@@ -3,7 +3,7 @@
 _GenericName=Spreadsheet
 _Comment=Calculation, Analysis, and Visualization of Information
 Exec=gnumeric %F_OR_U
-Icon=gnome-gnumeric
+Icon=gnumeric
 Terminal=false
 Type=Application
 Categories=Application;Office;Spreadsheet;Science;Math;GNOME;GTK;
