--- gnumeric-1.6.1/Makefile.in.desktop	2005-11-14 07:18:15.000000000 +0100
+++ gnumeric-1.6.1/Makefile.in	2005-11-26 22:53:14.000000000 +0100
@@ -344,18 +344,17 @@
 gnew_in_files = gnumeric.desktop.in
 gnew_DATA = gnumeric.desktop
 Mimedir = $(datadir)/mime-info
-Mime_DATA = gnumeric.keys gnumeric.mime
+Mime_DATA = gnumeric.mime
 @WITH_GNOME_TRUE@serverdir = $(libdir)/bonobo/servers/
 @WITH_GNOME_TRUE@server_in_files = GNOME_Gnumeric.server.in
 @WITH_GNOME_TRUE@server_DATA = GNOME_Gnumeric.server
 CLEANFILES = $(server_DATA) $(server_in_files) \
-	gnumeric.desktop gnumeric.keys
+	gnumeric.desktop
 
 DISTCLEANFILES = intltool-extract intltool-merge intltool-update
 EXTRA_DIST = \
 	README HACKING NEWS BEVERAGES BUGS MAINTAINERS AUTHORS $(change_logs)	\
 	gnumeric.desktop.in	\
-	gnumeric.keys.in	\
 	gnumeric.mime		\
 	gnumeric.spec		\
 	gnumeric.xsd		\
@@ -902,10 +901,6 @@
 
 @INTLTOOL_SERVER_RULE@
 
-gnumeric.keys: gnumeric.keys.in Makefile
-	sed -e 's^\@gnumeric_icondir\@^$(gnumeric_icondir)^g' < $(srcdir)/gnumeric.keys.in > gnumeric.keys.tmp \
-	&& mv gnumeric.keys.tmp gnumeric.keys 
-
 gnumeric.desktop: gnumeric.desktop.in Makefile $(INTLTOOL_MERGE) $(POFILES_FULL)
 	 LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@.tmp
 @WITH_GNOME_TRUE@	sed -e 's/F_OR_U/U/' <$@.tmp >$@.tmp2
