diff -up gnumeric-1.8.1/configure.in~ gnumeric-1.8.1/configure.in
--- gnumeric-1.8.1/configure.in~	2008-01-25 21:43:09.000000000 +0100
+++ gnumeric-1.8.1/configure.in	2008-01-25 21:43:09.000000000 +0100
@@ -429,7 +429,7 @@ if test "$try_gda" = "true"; then
 		[gda_msg=yes],
 		[gda_msg="NO.  libgda problem"])
 	if test "$gda_msg" = "yes"; then
-		PKG_CHECK_MODULES(GNOMEDB, [libgnomedb-3.0 >= 3.0.1],
+		PKG_CHECK_MODULES(GNOMEDB, [libgnomedb-3.0 >= 3.0.0],
 			[gnomedb_msg="yes"],
 			[gnomedb_msg="NO. libgnomedb problem"])
 		if test "$gnomedb_msg" = "yes"; then
diff -up gnumeric-1.8.1/configure~ gnumeric-1.8.1/configure
--- gnumeric-1.8.1/configure~	2008-01-25 21:43:04.000000000 +0100
+++ gnumeric-1.8.1/configure	2008-01-25 21:43:04.000000000 +0100
@@ -23352,12 +23352,12 @@ if test -n "$PKG_CONFIG"; then
         pkg_cv_GNOMEDB_CFLAGS="$GNOMEDB_CFLAGS"
     else
         if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"libgnomedb-3.0 >= 3.0.1\"") >&5
-  ($PKG_CONFIG --exists --print-errors "libgnomedb-3.0 >= 3.0.1") 2>&5
+    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"libgnomedb-3.0 >= 3.0.0\"") >&5
+  ($PKG_CONFIG --exists --print-errors "libgnomedb-3.0 >= 3.0.0") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_GNOMEDB_CFLAGS=`$PKG_CONFIG --cflags "libgnomedb-3.0 >= 3.0.1" 2>/dev/null`
+  pkg_cv_GNOMEDB_CFLAGS=`$PKG_CONFIG --cflags "libgnomedb-3.0 >= 3.0.0" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -23370,12 +23370,12 @@ if test -n "$PKG_CONFIG"; then
         pkg_cv_GNOMEDB_LIBS="$GNOMEDB_LIBS"
     else
         if test -n "$PKG_CONFIG" && \
-    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"libgnomedb-3.0 >= 3.0.1\"") >&5
-  ($PKG_CONFIG --exists --print-errors "libgnomedb-3.0 >= 3.0.1") 2>&5
+    { (echo "$as_me:$LINENO: \$PKG_CONFIG --exists --print-errors \"libgnomedb-3.0 >= 3.0.0\"") >&5
+  ($PKG_CONFIG --exists --print-errors "libgnomedb-3.0 >= 3.0.0") 2>&5
   ac_status=$?
   echo "$as_me:$LINENO: \$? = $ac_status" >&5
   (exit $ac_status); }; then
-  pkg_cv_GNOMEDB_LIBS=`$PKG_CONFIG --libs "libgnomedb-3.0 >= 3.0.1" 2>/dev/null`
+  pkg_cv_GNOMEDB_LIBS=`$PKG_CONFIG --libs "libgnomedb-3.0 >= 3.0.0" 2>/dev/null`
 else
   pkg_failed=yes
 fi
@@ -23394,9 +23394,9 @@ else
         _pkg_short_errors_supported=no
 fi
         if test $_pkg_short_errors_supported = yes; then
-	        GNOMEDB_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "libgnomedb-3.0 >= 3.0.1"`
+	        GNOMEDB_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "libgnomedb-3.0 >= 3.0.0"`
         else
-	        GNOMEDB_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "libgnomedb-3.0 >= 3.0.1"`
+	        GNOMEDB_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "libgnomedb-3.0 >= 3.0.0"`
         fi
 	# Put the nasty error message in config.log where it belongs
 	echo "$GNOMEDB_PKG_ERRORS" >&5
