--- gnumeric-1.4.1.orig/configure.in	2004-12-10 15:06:14.000000000 +0000
+++ gnumeric-1.4.1/configure.in	2004-12-15 13:54:19.899301960 +0000
@@ -708,7 +708,7 @@
 dnl * Check for Perl
 dnl **************************************************
 dnl
-perl_msg=no
+perl_msg="yes"
 AC_ARG_WITH(perl, [--{with,without}-perl   Compile the Perl plugin loader])
 AC_ARG_VAR(PERL, [The Perl executable.])
 if test "x$with_perl" != xno; then
